commands.js

```diff
@@ -1768,6 +1768,52 @@ exports.commands = {
 	slowchathelp: ["/slowchat [number] - Sets slowchat in the room for [number] seconds. Must be between 2-60 seconds. Requires @ * # & ~",
 		"/slowchat [off/disable] - Disables slowchat in the room. Requires @ * # & ~"],
 
+	stretching: function (target, room, user) {
+		if (!target) return this.sendReply("Stretching in this room is currently set to: " + (room.stretching ? room.stretching : false));
+		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (!this.can('editroom', null, room)) return false;
+
+		if (target === 'enable' || target === 'on') {
+			if (room.stretching) return this.errorReply("Stretching is already enabled in this room.");
+			room.stretching = true;
+		} else if (target === 'disable' || target === 'off') {
+			if (!room.stretching) return this.errorReply("Stretching is already disabled in this room.");
+			room.stretching = false;
```

Comment:
`.stretching` and `.capitals` seem too ambiguous as attributes.

I'd propose `.filterStretching` and `.filterCaps`